tasks/configure_filesystems.yml

```diff
@@ -59,3 +59,10 @@
     snapdir: "{{ fs.attributes.snapdir |default(zfs_storage_defaults.snapdir) |default(zfs_base_defaults.snapdir) }}"
     sync: "{{ fs.attributes.sync |default(zfs_storage_defaults.sync) |default(zfs_base_defaults.sync) }}"
     xattr: "{{ fs.attributes.xattr |default(zfs_storage_defaults.xattr) |default(zfs_base_defaults.xattr) }}"
+
+    "com.sun:auto-snapshot": "{{ fs.attributes.com_sun_auto_snapshot |default(zfs_storage_defaults.com_sun_auto_snapshot) |default(zfs_base_defaults.com_sun_auto_snapshot) | lower }}"
```

Comment:
This won't work. The proper way is `foo | default(bar | default(baz))` instead of `foo | default(bar) | default(baz)`

Reply:
Of course it works, I tested it and the other attributes also work just fine with the two levels of defaults.